hub/core/transform/transform.py

```diff
@@ -130,7 +131,8 @@ def eval(
             data_in = get_dataset_with_zero_size_cache(data_in)
 
         target_ds = data_in if overwrite else ds_out
-        check_transform_ds_out(target_ds, scheduler)
+        if not skip_ok:
```

Comment:
the initial length check is different from skip_ok, maybe add a separate argument for this?

Reply:
fixed.